pkg/project/util.go

```diff
@@ -175,3 +181,27 @@ var intersection = apply(func(bs []Equatable, a Equatable) bool {
 var difference = apply(func(bs []Equatable, a Equatable) bool {
 	return !contains(bs, a)
 })
+
+var getBoolEnvVar = func(key string, fallback bool) (bool, error) {
+	value, exists := os.LookupEnv(key)
+	if exists {
+		boolValue, err := strconv.ParseBool(value)
+		if err == nil {
+			return boolValue, nil
```

Comment:
You could summarize this whole function yo the point of not needing it. When i get back to my laptop I'll show you.